subprojects/jfinkhaeuser-json-rpc.wrap

```diff
@@ -0,0 +1,8 @@
+[wrap-file]
+directory = json-rpc
+source_url = https://codeberg.org/jfinkhaeuser/json-rpc/archive/v0.1.1.tar.gz
+source_filename = v0.1.1.tar.gz
+source_hash = 88002cc87ae87d56c76e9e4bc46456198cb4dd048913d4395ee79c974e14df8d
+
+[provide]
+json_rpc = json_rpc_dep
```

Comment:
It seems that this provides a subproject accessible declares dependency, but does not install a pkg-config file for system use or for use in projects that use a build system other than meson.

Reply:
Correct. I don't particularly mind pkg-config, but it's not very high on my priority list.

Reply:
It's trivial to add, meson has a built-in exporter for it. :)

```meson
pkgconfig = import('pkgconfig')

pkgconfig.generate(foo_lib)
```

There are optional kwargs that can fine tune the metadata, for example giving the project description.

Reply:
Just to note, adding a `.pc` file is not required. In fact if upstream does not provide one, then it makes sense to _not_ add one at all, because otherwise people will start to depend on it existing which will cause problems.